types/types.go

```diff
@@ -79,6 +79,15 @@ type Layer struct {
 	// https://github.com/opencontainers/image-spec/blob/8b9d41f48198a7d6d0a5c1a12dc2d1f7f47fc97f/specs-go/v1/mediatype.go
 	MediaType string `json:"mediatype"`
 	LayerPath string `json:"layer-path,omitempty"`
+	CreatedBy string `json:"created-by,omitempty"`
+	Author    string `json:"author,omitempty"`
+	Comment   string `json:"comment,omitempty"`
+}
+
+type Metadata struct {
```

Comment:
Do you think we could use https://github.com/opencontainers/image-spec/blob/3c3d71753a024375415026b918ba470714383a81/specs-go/v1/config.go#L74 instead?

Reply:
Yes. That's a much better idea. Done.